src/openssl.c

```diff
@@ -3403,11 +3403,20 @@ static BIO *getbio(lua_State *L) {
 } /* getbio() */
 
 
+static int pem_pw_cb(char *buf, int size, int rwflag, void *u) {
+	if (!u)
+		return 0;
+	char *pass = (char *) u;
+	strncpy(buf, pass, size);
+	return MIN(strlen(pass), (unsigned int) size);
+} /* pem_password_cb() */
```

Comment:
I guess there is a typo in the comment. Should be `/* pem_pw_cb() */`

Reply:
This still needs fixing